src/shipchain_common/test_utils/helpers.py

```diff
@@ -131,3 +132,18 @@ class GeoCoderResponse:
     def __init__(self, status, point=None):
         self.ok = status
         self.xy = point
+
+
+# pylint: disable=too-many-arguments, redefined-builtin
```

Comment:
Please google W0622 and see why this is dangerous. We don't want to redefine builtin methods. Whatever triggered this should be renamed.

Reply:
This actually wasn't supposed to be in this PR I thought I had removed it. I will address this in the appropriate PR.